core/src/main/scala/laserdisc/package.scala

```diff
@@ -122,4 +122,19 @@ package object laserdisc {
     def unapply(s: String): Option[Double] =
       try { Some(j.Double.parseDouble(s)) } catch { case _: NumberFormatException => None }
   }
+
+  implicit final class WidenOps1[F[_], A](private val fa: F[A]) extends AnyVal {
+    def widen[AA >: A]: F[AA] = fa.asInstanceOf[F[AA]]
+  }
+
+  implicit final class WidenOps2[F[_, _], A, B](private val fab: F[A, B]) extends AnyVal {
+    def widenLeft[AA >: A]: F[AA, B]                                                 = fab.asInstanceOf[F[AA, B]]
+    def widenRight[BB >: B]: F[A, BB]                                                = fab.asInstanceOf[F[A, BB]]
```

Comment:
Apologies I suggested this in the first place. The more I look at this the more I think we should use 
```scala
def widenLeft[AA](implicit ev: A <:< AA): F[AA, B]  = fab.asInstanceOf[F[AA, B]]
def widenRight[BB](implicit ev: B <:< BB): F[A, BB] = fab.asInstanceOf[F[A, BB]]
```
in this very case it's the same as we always provide `AA` and `BB` at call site (there's nothing to infer fortunately). But for the sake of consistency with the rest I would prefer using generalised type constraints instead of type bounds. The same for `WidenOps1`'s `widen` and `WidenOps3`'s `widenRight`.
What do you think ?